packages/sdk-codegen/src/python.gen.spec.ts

```diff
@@ -76,21 +82,30 @@ class LookerSDK(api_methods.APIMethods):
 `)
     })
     it('has a models epilogue', () => {
+      const type = apiTestModel.types.LookmlModelExploreJoins
+      gen.declareType(indent, type)
       expect(gen.modelsEpilogue('')).toEqual(`
 
 # The following cattrs structure hook registrations are a workaround
 # for https://github.com/Tinche/cattrs/pull/42 Once this issue is resolved
 # these calls will be removed.
 
 import functools  # noqa:E402
-from typing import Any
 try:
     from typing import ForwardRef  # type: ignore
 except ImportError:
     from typing import _ForwardRef as ForwardRef  # type: ignore
 
-structure_hook = functools.partial(sr.structure_hook, globals(), sr.converter)
-
+forward_ref_structure_hook = functools.partial(sr.forward_ref_structure_hook, globals(), sr.converter)
+translate_keys_structure_hook = functools.partial(sr.translate_keys_structure_hook, sr.converter)
+sr.converter.register_structure_hook(
+    ForwardRef("LookmlModelExploreJoins"),  # type: ignore
+    forward_ref_structure_hook  # type:ignore
+)
+sr.converter.register_structure_hook(
+    LookmlModelExploreJoins,  # type: ignore
+    translate_keys_structure_hook  # type:ignore
+)
```

Comment:
running `gen.declareType` above so that we can make sure we're registering the `translate_keys_structure_hook` on models. We weren't before and I think there was a silent bug with `LookmlModelExploreJoins` where `from_`/`"from"` was getting lost in translation...

the unittests passed because we explicitly register this structure hook on "Model"